app/src/main/java/nu/yona/app/ui/message/MessageStickyRecyclerAdapter.java

```diff
@@ -64,63 +71,137 @@ public MessageItemViewHolder onCreateViewHolder(ViewGroup parent, int viewType)
 	@Override
 	public void onBindViewHolder(MessageItemViewHolder holder, int position)
 	{
-		YonaMessage yonaObject = (YonaMessage) getItem(position);
+		currentYonaMessage = (YonaMessage) getItem(position);
+		currentMessageItemHolder = holder;
+		if (currentYonaMessage != null)
+		{
+			setUpMessageListItemTitle();
+			setUpMessageItemHolderDetails();
+			currentMessageItemHolder.deleteMsg.setTag(currentYonaMessage);
+			currentMessageItemHolder.messageContainer.setTag(currentYonaMessage);
+		}
+	}
 
-		if (yonaObject != null)
+	private void setUpMessageListItemTitle()
+	{
+		if (currentYonaMessage.getNotificationMessageEnum() != null && !TextUtils.isEmpty(currentYonaMessage.getNotificationMessageEnum().getUserMessage()))
```

Comment:
It should never be possible that ``getNotificationMessageEnum`` returns null. Assign ``NONE_NONE`` as default value in ``YonaMessage``, and you'll never need to check for null again.

Reply:
Done.